gwsumm/tabs/core.py

```diff
@@ -45,16 +45,20 @@
 
 from gwdetchar.io import html as gwhtml
 
+from .. import __version__
 from .. import html
```

Comment:
I would combine these two imports:

```suggestion
from .. import (__version__, html)
```